content/workshops/cascading-layouts.md

```diff
@@ -86,6 +107,8 @@ media:
     id: workshop-video
   - youtube: De6QMQA2aio
     id: workshop-video-2
+  - youtube: ccL4JP4f_R4
+    id: workshop-video-3
```

Comment:
@SondraE adding `span: full` to one of these (maybe the first one) will make that video fill the whole width. I think it would help with the layout a bit, and also give people a place to start?